src/couch_replicator_utils.erl

```diff
@@ -451,7 +451,12 @@ maybe_upgrade_wait(HttpDb) ->
 % DBCore doesn't use that feature. Here it is read directly from config
 % setting and used only for forward compatibility
 get_couch_server_uuid() ->
-    config:get("couchdb", "uuid").
+    case config:get("couchdb", "uuid") of
+        UUID when is_list(UUID) ->
+            ?l2b(UUID);
+        UUID ->
+            UUID
```

Comment:
What can this be other than a list?

Reply:
Could be undefined. Maybe we should explicitly match for `undefined` and list